platform-service-framework/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
 
   // Use for thread dump servlet
   implementation("io.dropwizard.metrics:metrics-servlets:4.1.16")
-  implementation("org.eclipse.jetty:jetty-servlet:9.4.39.v20210325")
+  implementation("org.eclipse.jetty:jetty-servlet:9.4.42.v20210604")
```

Comment:
Can you update this build scritp too - `/platform-metrics/build.gradle.kts` - for jetty version?